include/bitboard.hpp

```diff
@@ -63,12 +63,26 @@ class Bitboard {
     // Bitwise NOT
     friend constexpr Bitboard operator~(const Bitboard& b) { return ~b; }
 
+    // Prints the bitboard in a 8x8 grid format ; 1 = piece, 0 = empty square
```

Comment:
I don't think this comment is necessary, beyond the `1 = piece, 0 = empty square` part, and that can go inside the body of the function definition.

Reply:
done